packages/falter-berlin-service-registrar/doc/falter-berlin-service-registrar.rst

```diff
@@ -0,0 +1,72 @@
+
+
+LuCI-App (Webinterface)
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Der Service-Registrar befindet sich in der Admin-Oberfläche unter *Freifunk* -> *Freifunkdienste registrieren*. In der App können entweder Webseiten oder Allgemeine Dienste angelegt werden.
+
+Webeiten sind Dienste, bei denen der Freifunk-Router selbst eine (statische) Webseite ausliefert. Es wird also nicht nur entsprechend OLSR konfiguriert, sondern auch der Webserver auf dem Freifunkrouter so eingestellt, dass er die Webseite ausliefert.
+
+Allgemeine Dienste sind etwas anders. Sie eignen für Dienste, die auf einem anderen Gerät im lokalen Freifunknetzwerk laufen. Wenn man z.B. eine Webcam bereitstellen möchte, oder irgendeine andere Art von Server hat, eignet sich diese Art von Diensten am besten dafür.
+
+Am besten sollte die IP-Adresse des entsprechenden Gerätes dann auf statisch gestellt werden (unter *Status* -> *Übersicht* und dann im Abschnitt *Aktive DHCP-Leases* auf den Knopf *Auf statisch setzen* drücken).
+
+Die benötigten Felder sind sind eigentlich relativ selbsterklärend. Nachdem die Änderungen mit dem Knopf *Speichern & Anwenden* abgespeichert wurden, beginnt der Freifunkrouter mit der Einrichtung.
+
+Bis der Service im kompletten Freifunknetz verfügbar ist, können bis zu 5 Minuten vergehen.
+
+
+Kommandozeile
+^^^^^^^^^^^^^
+
+Um einen Dienst über die Kommondozeile einzutragen, muss zuerst die Konfigurationsdatei unter ``/etc/config/ffservices`` angepasst werden und anschließend das Skript ``register-services`` ausgeführt werden.
+
+Je nachdem, ob man eine Webseite oder einen Allgemeinen Dienst eintragen möchte, gibt es zwei unterscheidliche Typen von UCI-Sections. Einmal den Typen ``website`` und einmal den Typen ``service``. Beide Typen unterscheiden sich geringfügig in ihren Feldern:
+
+..
+    config website "testsite"
+            option fqdn         "testsite.olsr"
+            option description  "Webseite auf dem lokalen Router"
+            option protocol     "tcp"
+            option port         "81"
+            option web_root     "/tmp/www/testsite"
+            option disabled     0
+
+..
+    config service "extern"
+            option fqdn         "extern.olsr"
+            option description  "Ein Dienst der auf einem anderen Geraet lauft"
+            option protocol     "udp"
+            option port         "80"
+            option ip_addr      "10.36.0.34"
+            option disabled     0
+
+Besondere Vorsicht braucht es bei dem Feld ``description``: Dort sollten keine Sonderzeichen, wie z.B. Punkte``.``, Kommata``,``, Dollarzeichen ``$``, usw. verwendet werden.
+
+Die IP-Adresse im Feld ``ip_addr`` muss aus dem Host-Network des eigenen Routers stammen ("DHCP-Adressen" aus der IP-Email).
+
+Nachdem die Datei entsprechend angepasst wurde, werden die Dienste mit einem Aufruf von ``register-services`` eingetragen. Dienste, die vorher von Hand im OLSR-Daemon eingetragen wurden, werden dabei entfernt.
```

Comment:
Gibt es hierfür einen Hinweis? Falls nicht sollte man das in der Dokumentation schreiben "ohne Hinweis entfernt".

Reply:
Hinweis in neuer Version hinzugefügt.